WordPress/src/main/java/org/wordpress/android/util/DateTimeExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.util
+
+import java.text.DateFormat
+import java.util.Date
+import java.util.Locale
+
+fun Date.toFormattedDateString(): String {
```

Comment:
good example of an extension function 👍. Maybe add a comment for future generations?

Reply:
What kind of comment do you have in mind? Is the code not self-explanatory? :)

Reply:
it probably is 😄 and when I think about it, the resulting string is probably different for every Locale